src/main/resources/model.ttl

```diff
@@ -132,6 +132,9 @@ rm:token rdf:type owl:DatatypeProperty .
 <http://xmlns.com/foaf/0.1/mbox> rdf:type owl:DatatypeProperty .
 
 
+###  http://onto.fel.cvut.cz/ontologies/record-manager/has-reject-message
+rm:has-reject-message rdf:type owl:DatatypeProperty .
```

Comment:
We have a name-ing convention to not use the `has-` prefix for `owl:DatatypeProperty`-ies.

Thus i suggest (see also comment above):
rm:has-reject-message --> rm:reject-reason

Reply:
done